js/init.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+define(function(require) {
+	var App = require('app');
+
+	$(function() {
+		// Start app when the page is ready
+		console.log('starting Mail…');
```

Comment:
before someone nitpicks this as debug code – I've added in on purpose because it could be handy to debug js errors in the future if someone reports a weird client error. We'd then know whether the app tried to start or not.